app/views/ProjectsView/components/AddEditProjectDialog/index.tsx

```diff
@@ -139,17 +129,28 @@ const AddEditProjectDialog = ({
         // Return the updated projects
         const updatedProjects = await api.get(`/project?admin=${adminAccess}`).request();
         onClose(updatedProjects, Boolean(!editData));
-      } catch (e) {
-        snackbar.error(`Error ${editData ? 'editing' : 'creating'} project, ${e?.content?.error?.message}`);
-      } finally {
-        setIsSaving(false);
-      }
+        queryClient.invalidateQueries(['projectReports']);
```

Comment:
I don't think this will invalidate `queryKey: ['projectReports', editData?.ident]`, you'd need the full key

Had `projectReports` been used somewhere else as well and you want it to invalidate everything, this would have been good, but since it's just limited to a single ident, it's a waste of API resources, and I don't think the `queryClient` stored this query as well anyways